docs/adding_standalone_benchmark.md

```diff
@@ -0,0 +1,58 @@
+# Adding Standalone Benchmarks in vSwarm
+This is a detailed document of how to add a standalone benchmark in vSwarm.
```

Comment:
not sure why "standalone", is it different for multi-function apps?

Reply:
Yes, this document is just for adding standalone benchmark, because multi-function apps might have different development  procedure, for example, vSwarm-proto only contains protobuf for standalone functions, and multi-function benchmark doesn't rely on the relay docker instance, etc.